src/components/ui/Button/button.scss

```diff
@@ -165,12 +166,11 @@
   }
 
   &[data-fs-button-size="small"] {
-    column-gap: var(--fs-button-small-gap);
+    --fs-control-tap-size: var(--fs-button-small-min-height);
```

Comment:
Is it being used?

Reply:
`--fs-control-tap-size` is a global token